pkg/scheduler/framework/plugins/nodevolumelimits/utils.go

```diff
@@ -47,6 +47,10 @@ func isCSIMigrationOn(csiNode *storagev1.CSINode, pluginName string) bool {
 		if !utilfeature.DefaultFeatureGate.Enabled(features.CSIMigrationAWS) {
 			return false
 		}
+	case csilibplugins.PortworxVolumePluginName:
+		if !utilfeature.DefaultFeatureGate.Enabled(features.CSIMigrationPortworx) {
```

Comment:
Scheduler changes LGTM.

Will approve once sig-storage approves the feature gate's addition.